code/game/objects/items/weapons/circuitboards/machinery/power.dm

```diff
@@ -28,4 +28,16 @@
 	req_components = list(
 		/obj/item/weapon/cell/large = 3
 	)
-	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_GLASS = 3)
\ No newline at end of file
+	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_GLASS = 3)
+
+/obj/item/weapon/circuitboard/antigrav
+	name = T_BOARD("antgrav generator")
+	build_path = /obj/machinery/antigrav
+	board_type = "machine"
+	origin_tech = list(TECH_ENGINEERING = 4, TECH_MAGNET = 4)
+	req_components = list(
```

Comment:
these components don't really make much sense, what would a grav blocker need with a matter bin?
 did you copy them from something else?

It'd be better to use subspace components like ansible crystal